packages/@o3r/schematics/src/tasks/package-manager/add-dev-dependency.ts

```diff
@@ -27,7 +29,7 @@ export class AddDevInstall extends NodePackageInstallTask {
       options: {
         ...defaultConfig.options,
         command: 'install',
-        packageName: `${this.packageName!} ${this.packageManager === 'yarn' ? '--prefer-dev' : '--save-dev'}`
+        packageName: `${this.packageName!} ${this.packageManager === 'yarn' ? '--prefer-dev' : '--save-dev'}${this.force && this.packageManager === 'name' ? ' --force' : ''}`
```

Comment:
this.packageManager === 'npm' ?

Reply:
Oooooh, this one is a huge mistake.
I guess a autocomplete of VsCode somehow.
This is why our own supported `PackageManger` type can help :D